src/bin/local.rs

```diff
@@ -183,13 +206,33 @@ fn main() {
         }
     };
 
+    #[cfg(feature = "dns-relay")]
+    {
+        if let Some(local_dns_addr) = matches.value_of("LOCAL_DNS_ADDR") {
+            let addr: SocketAddr = local_dns_addr.parse().expect("local dns address");
+            config.local_dns_addr = Some(addr);
+        }
+
+        if let Some(remote_dns_addr) = matches.value_of("REMOTE_DNS_ADDR") {
+            let addr: SocketAddr = remote_dns_addr.parse().expect("remote dns address");
+            config.remote_dns_addr = Some(addr);
+        }
+
+        if let Some(dns_relay_addr) = matches.value_of("DNS_RELAY_ADDR") {
+            let addr: SocketAddr = dns_relay_addr.parse().expect("dns relay address");
+            config.dns_relay_addr = Some(addr);
+        }
+    }
+
     if cfg!(target_os = "android") {
+        config.local_dns_path = Some("local_dns_path".to_string());
```

Comment:
This seems unused. Maybe use this on `cfg!(target_os = "android")`? I don't see the necessity of exposing another port.

Reply:
Yes, this is not used anywhere in code. What is this for? @madeye

Reply:
it's for Mygod's UNIX domain socket based local resolver.

I'll apply `cfg!(target_os = "android")` here.

BTW, I think local socket with UDP (in kernel, it's also implemented like UNIX socket) is already good enough, maybe UNIX socket is not necessary?